lib/dora-parser/src/parser.rs

```diff
@@ -289,7 +289,7 @@ impl<'a> Parser<'a> {
             name: ident,
             pos: pos,
             has_open: has_open,
-            internal: internal,
+            internal: builtin,
```

Comment:
When we don't use `internal` anymore, I think we should sooner or later also rename this field. But no need to do this in this commit.

Reply:
+1. Seems like an oversight.